libvirt/resource_libvirt_domain_test.go

```diff
@@ -1767,3 +1767,58 @@ func testAccCheckLibvirtDomainDestroy(s *terraform.State) error {
 	}
 	return nil
 }
+func TestAccLibvirtDomain_FwCfgName(t *testing.T) {
```

Comment:
can we have a comment here after this line specifying the `why` of this test exist? I would just state it in high manner that we want to support other things then coreos and for that we need to test that fwcfname (what a cryptic name ! ) . So i would like some comment which will help us to maintain in future

Reply:
I just want at the end to have a highlevel comment, to say what we are testing and why.